nvml-wrapper/src/device.rs

```diff
@@ -624,6 +624,45 @@ impl<'nvml> Device<'nvml> {
         }
     }
 
+    /**
+    Gets information about processes with a compute context running on this `Device`.
+    Note that processes list can differ between the accounting call and the list gathering
+
+    # Errors
+
+    * `Uninitialized`, if the library has not been successfully initialized
+    * `InvalidArg`, if this `Device` is invalid
+    * `GpuLost`, if this `Device` has fallen off the bus or is otherwise inaccessible
+    * `Unknown`, on any unexpected error
+
+    # Device Support
+
+    Supports Volta or newer fully supported devices.
+    */
+    #[doc(alias = "nvmlDeviceGetiMPSComputeRunningProcesses_v3")]
+    pub fn mps_running_compute_processes(&self) -> Result<Vec<ProcessInfo>, NvmlError> {
+        let sym = nvml_sym(
+            self.nvml
+                .lib
+                .nvmlDeviceGetMPSComputeRunningProcesses_v3
+                .as_ref(),
+        )?;
+
+        unsafe {
+            let mut hints: c_uint = 0;
+
+            nvml_try(sym(self.device, &mut hints, ptr::null_mut()))?;
+
+            hints.checked_shr(1);
```

Comment:
This seems like a typo. Did you mean to do this instead?
```rs
hints = hints.checked_shr(1);
```

Reply:
No I just forgot to treat this part. You re almost right, since it s the checked flavor it  Option<Self> return type